tests/unit/src/test/scala/tests/codeactions/OrganizeImportsLspSuite.scala

```diff
@@ -0,0 +1,112 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.codeactions.OrganizeImports
+
+class OrganizeImportsLspSuite
+    extends BaseCodeActionLspSuite("OrganizeImports") {
+  val kind: String = OrganizeImports.kind
+  val scalacOption: List[String] = List("-Ywarn-unused-import")
+  val scalafixConf: String =
+    """|/.scalafix.conf
+       |rules = [
+       |  OrganizeImports,
+       |  ExplicitResultTypes,
+       |  RemoveUnused
+       |]
+       |
+       |ExplicitResultTypes.rewriteStructuralTypesToNamedSubclass = false
+       |
+       |RemoveUnused.imports = false
+       |
+       |OrganizeImports.groupedImports = Explode
+       |OrganizeImports.expandRelative = true
+       |OrganizeImports.removeUnused = true
+       |OrganizeImports.groups = [
+       |  "scala."
+       |  "re:javax?\\."
+       |  "*"
+       |]
+       |
+       |""".stripMargin
+
+  check(
+    "basic organize imports",
```

Comment:
we generally keep test names short and without spaces, so that they're easier to target with a test filter. So something like:

```suggestion
    "basic",
```

and so on